unit_tests/kernels/UnitTestKernelUtils.C

```diff
@@ -1376,7 +1377,7 @@ void calc_projected_nodal_gradient_boundary(
       auto v_dnv = preReqData.get_scratch_view_1D(dnv);
       auto v_vector = preReqData.get_scratch_view_2D(vectorField);
       auto v_scs_areav = preReqData.get_me_views(sierra::nalu::CURRENT_COORDINATES).scs_areav;
-      const ngp::Mesh::ConnectedNodes node_rels = preReqData.elemNodes;
+      const stk::mesh::NgpMesh::ConnectedNodes node_rels = preReqData.elemNodes;
```

Comment:
I am really liking `auto` right now :)

Reply:
Normally I'm against auto, but you might be right on this one.  :-)